CHANGES.rst

```diff
@@ -213,6 +213,8 @@ Bug fixes
 
 - ``astropy.wcs``
 
+  - Fix for allowing WCS to take Image HDU. [#4493]
```

Comment:
This is an API change. Just write "WCS objects can now be initialized with an ``ImageHDU`` or ``PrimaryHDU`` object. [#4493]"